README.md

```diff
@@ -6,6 +6,7 @@ This repository contains the following crates:
 | --- | --- | --- |
 | [db-allocator](crates/db-allocator) | allocator for vmm resource | TBD |
 | [db-arch](crates/db-arch) | collections of CPU architecture related modules | TBD |
+| [db-boot](crates/db-boot) | collections of constants and utilities used in VM boot stage | TBD |
```

Comment:
collections of constants, structs and utilities used during VM boot stage?

Reply:
Changed